tests/src/test/scala/cats/tests/CategoryTests.scala

```diff
@@ -1,7 +1,7 @@
 package cats
 package tests
 
-import cats.kernel.laws.GroupLaws
+import cats.kernel.laws.discipline.{MonoidTests => MonoidTypeclassTests}
```

Comment:
Another nit: Maybe we can call these `MonoidLawTests`, because the other `MonoidTest` is also testing the Type Class.